### ISSUE TITLE
Focus style on links do not work on Windows High contrast mode

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#2863]
### Description of the bug

When using Windows high contrast mode. Windows will overwrite the colors used on our components. On most of our components this works as expected, but with our link that has a different focus state, its not working. We should implement a fallback for this scenario. 

Seems like outline is used by Windows when using high contrast mode. It might be that because link do not have an outline on focus that causing the issue. 

> ...so outline remains as the only reliable way to apply a focus state on an element in WHCM. Always keep that in mind: if you’re going to use something different than an outline to highlight a focus state in an element, add the property outline-color: transparent as a fallback to not missing a focus state in this mode.

[source](https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/)

### Steps To Reproduce

Test our links using a keyboard and windows high contrast mode turned on

**Comment by mimarz** [comment]
Any thoughts on how to solve this @eirikbacker ?

Maybe we can have a transparent outline so that windows can color this for contrast mode?

**Comment by eirikbacker** [comment]
Sounds like it :) but I we need a windows user/machine to test this - is this something you can look into @Barsnes ? <3

**Comment by Barsnes** [comment]
> Sounds like it :) but I we need a windows user/machine to test this - is this something you can look into @Barsnes ? <3

As long as this bug is present in windows 11, I can test this, yes

**Comment by Barsnes** [comment]
I am testing using windows 11 high contrast with the theme "Nattehimmel", but the issues seem to perisist between the different themes.

This is a link with `:focus-visible`:
![image](https://github.com/user-attachments/assets/c3494272-ea42-47e1-8d21-b827b6777682)

I did also find that a lot of things that don't work:
Can't see selected radio button
![image](https://github.com/user-attachments/assets/7ef1e912-1526-408f-840a-15df4ca512e1)
How it looks with an unstyled:
![image](https://github.com/user-attachments/assets/1ab945ad-ac8d-4b58-872b-34ab311deaf7)

---

Can't see selected checkbox:
![image](https://github.com/user-attachments/assets/8105839d-9ef0-469f-9f1d-314b434501dc)
How it looks with an unstyled:
![image](https://github.com/user-attachments/assets/3f11169b-e4e9-4c2b-910c-090c9a3bed04)
